packages/firestore/src/api/database.ts

```diff
@@ -398,14 +398,30 @@ export class Firestore implements firestore.FirebaseFirestore, FirebaseService {
           'any other methods on a Firestore object.'
       );
     }
+
+    let synchronizeTabs = false;
+
+    if (settings) {
+      if (settings.experimentalTabSynchronization !== undefined) {
+        log.error(
+          'The `experimentalTabSynchronization` setting has been renamed to ' +
+            '`synchronizeTabs`. In a future release, the setting  will be removed ' +
+            'and it is recommended that you remove it from your ' +
+            'firestore.enablePersistence() call now.'
+        );
+      }
+      synchronizeTabs = objUtils.defaulted(
+        settings.synchronizeTabs !== undefined
+          ? settings.synchronizeTabs
+          : settings.experimentalTabSynchronization,
+        DEFAULT_SYNCHRONIZE_TABS
```

Comment:
Way to go naming the default correctly ahead of time!

Reply:
Why do you assume the API proposal wasn't based on this name? :)